user/awesome/template.py

```diff
@@ -0,0 +1,59 @@
+pkgname = "awesome"
```

Comment:
look at any other template in this repository and copy the style. all of these newlines should not be here, and the order of everything is hence incorrect